### ISSUE TITLE
The concat var isn't save as it should.

### ISSUE THREAD

**Issue by fontinixxl** [fontinixxl/moodle-qtype_programmedresp#19]
Once a concat var is generated we have the choice to rename it. If it does not renamed (we will use the proposed concatvar_x name, where 'x' is a ordered number) the name is stored on DB without the 'x' number.

**Comment by fontinixxl** [comment]
After that I check that this issue produce a Warning and notice debug message when a question is editing.
```

Notice: Trying to get property of non-object in /var/www/html/question/type/programmedresp/programmedresp_output.class.php on line 238

Notice: Trying to get property of non-object in /var/www/html/question/type/programmedresp/programmedresp_output.class.php on line 238

Notice: Undefined index: _ in /var/www/html/question/type/programmedresp/programmedresp_output.class.php on line 238
```

**Comment by fontinixxl** [comment]
L'error es produeix quan intenta assignar una variable linker a un argument (previament seleccionat) però NO ens trobem dins d'un context quiz o bé després de FER restore, que NO tenim les dades de `programmedresp_v_arg`.

**Comment by fontinixxl** [comment]
Fixed #23 too!
